README.md

```diff
@@ -1 +1,63 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
+
+
+## Important note
+
+We do not have a working demo yet. [we](https://github.com/FunnyGuilds/FunnySectors/graphs/contributors) are actively working on creating one.
+
+
+## FAQ
+
+#### What are the priorities for the project?
+
+Well... We need a working demo. That is the most important thing for now. As [bopke](https://github.com/bopke) (project leader) [mentioned](https://github.com/FunnyGuilds/FunnySectors/issues/6) we need a working demo to avoid burnout.
+
+#### What technology are you going to start with?
+
+* Project language: [Java](https://www.java.com/en/)
+* Test language: [Kotlin](https://kotlinlang.org/)
+* Database: [MySql](https://www.mysql.com/)
+* Message broker: [Redis](https://redis.io/)
+* Minecraft platform: [Spigot](https://www.spigotmc.org/)
```

Comment:
I'd rather say that paperspigot is better target that spigot, we may had overlooked that before

Reply:
Do you really believe, that some 1.8 servers will be running paperspigot? NO! In fact I can best that the majority of our 1.8 target is using some crazy proprietary spigot fork. We can discuss it in the platform support issue

Reply:
I'm not sure how additional paper api looks like at 1.8 because I'm not really happy with supporting such a old version, is it really that wild in 1.8?

Reply:
Well... you agreed to support 1.8, so we can't just relay on people having paperspigot. There are a lot of custom forks for spigot on that version. I do think that we can depend on paper on 1.16+ but doing it on 1.8 is going to drastically reduce our end target.